package.json

```diff
@@ -112,13 +110,13 @@
     "@apexdevtools/apex-parser": "4.4.0",
     "antlr4ts": "0.5.0-alpha.4",
     "data-structure-typed": "^2.0.4",
+    "vscode-languageserver-protocol": "^3.17.5",
     "vscode-languageserver-textdocument": "^1.0.12",
     "vscode-uri": "^3.1.0"
   },
   "private": true,
   "workspaces": [
-    "packages/*",
```

Comment:
I think we'll need to keep playwright in the devdependencies and the e2e-tests workspace